<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plotView</class>
 <widget class="QWidget" name="plotView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot View</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_saveFigure">
     <property name="text">
      <string>Save figure</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_text">
     <property name="text">
      <string>interval = </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
